app/controllers/api/v1/onesignal_controller.rb

```diff
@@ -0,0 +1,21 @@
+module API
+  module V1
+    class OnesignalController < ApplicationController
+      def send_notification
+        onesignal_service = OneSignalService.new(
+          '93cf85a7-3498-4bb6-b949-0192c66835dc',
+          'ODkwMTc4MzYtM2Q2MC00ZTRiLThlODktMmMwYjkzYTYzYmYx'
```

Comment:
We need to store this in credentials file https://github.com/amprotocolab/internal-optigate-api/blob/main/config/credentials.yml.enc and use that to encrypt decrypt this